main/src/com/google/refine/model/medadata/MetadataFactory.java

```diff
@@ -11,4 +33,46 @@ public static IMetadata buildMetadata(MetadataFormat format) {
         
         return metadata;
     }
+    
+    /**
+     * build an empty Data Package Metadata
+     * @return
+     */
+    public static DataPackageMetadata buildDataPackageMetadata() {
+        return (DataPackageMetadata) buildMetadata(MetadataFormat.DATAPACKAGE_METADATA);
+    }
+    
+    /**
+     * Build an empty data package metadata, then populate the fields from the Project Metadata
```

Comment:
Sorry but again, that is not a solution - what if the metadata changes later on? For instance:
* `buildDataPackageMetadata` is called
* the user edits the schema
* the user edits project metadata
=> the data package metadata is out of sync from the project metadata

At this stage, calling `buildDataPackageMetadata` again would discard the changes made to the schema.

Again, the clean solution for this is just not to duplicate this metadata in the first place.